src/gui/networksettings.cpp

```diff
@@ -177,6 +181,21 @@ void NetworkSettings::loadBWLimitSettings()
     _ui->uploadSpinBox->setValue(cfgFile.uploadLimit());
 }
 
+void NetworkSettings::loadMeteredSettings()
+{
+    if (QNetworkInformation *qNetInfo = QNetworkInformation::instance()) {
```

Comment:
The change only takes effect on the next connection change/client restart.
Should we improve that?

Reply:
Why? Other settings are also loaded once.